.github/workflows/containers.yml

```diff
@@ -76,7 +76,7 @@ jobs:
               tags.push('latest');
 
               // If on a release commit add `v7`, `v7.1` and `v7.1`
-              if (${{ toJson(github.event.head_commit.message) }}.startsWith('release:')) {
+              if (${{ toJson(github.event.head_commit.message) }}.startsWith('chore: release')) {
```

Comment:
would be best to keep the `release:` commit message so we have a consistent history, I think you can change it in release-please settings.